srcpkgs/arcan/template

```diff
@@ -1,45 +1,54 @@
 # Template file for 'arcan'
 pkgname=arcan
-version=0.5.5.2
-revision=3
+version=0.6.0
+revision=1
 build_wrksrc=src
 build_style=cmake
-configure_args="-DDISTR_TAG='Void Linux' -DVIDEO_PLATFORM=egl-dri"
-makedepends="ffmpeg-devel harfbuzz-devel liblzma-devel MesaLib-devel
+configure_args="-DDISTR_TAG='Void Linux' -DVIDEO_PLATFORM=egl-dri
+ $(vopt_if luajit '-DDISABLE_JIT=OFF' '-DDISABLE_JIT=ON')"
+hostmakedepends="pkg-config"
+makedepends="MesaLib-devel ffmpeg-devel file-devel freetype-devel liblzma-devel
  libopenal-devel libusb-devel libvncserver-devel libxkbcommon-devel sqlite-devel
- $(vopt_if tesseract 'tesseract-ocr-devel leptonica-devel') vlc-devel"
+ vlc-devel
+ $(vopt_if tts 'libespeak-devel')
+ $(vopt_if luajit 'LuaJIT-devel' 'lua51-devel')
+ $(vopt_if tesseract 'tesseract-ocr-devel leptonica-devel') 
+ $(vopt_if wayland 'wayland-devel wayland-protocols libxcb-devel xcb-util-wm-devel')
+ "
 short_desc="Combined display server, multimedia framework and game engine"
 maintainer="Piraty <piraty1@inbox.ru>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later, BSD-3-Clause"
 homepage="https://arcan-fe.com/"
 _versionOpenal=0.5.4
 distfiles="https://github.com/letoram/${pkgname}/archive/${version}.tar.gz
  https://github.com/letoram/openal/archive/${_versionOpenal}.tar.gz>openal_arcan.${_versionOpenal}.tar.gz"
-checksum="a5f775789e22f6a4f16742ae9375bb5b7999aab83398adeab2bdc3f70c321cc0
+checksum="7b7aeb0fa6b89b7e3779d2e078b626566bdfa86c756d7ea1cc4f84b9e60d16a0
  3a50a87c05b67c466a868cc77f8dc7f9cfc9466aeeafcd823daca0d108c504da"
+patch_args="-Np1"
 
-if [ "$XBPS_TARGET_WORDSIZE" != "$XBPS_WORDSIZE" ]; then
-	configure_args+=" -DDISABLE_JIT=ON"
-	makedepends+=" lua51-devel"
-else
-	makedepends+=" LuaJIT-devel"
+build_options="luajit tesseract tts wayland"
+desc_option_tts="Enable support for text-to-speech via espeak"
+desc_option_tesseract="Enable support for OCR via tesseract"
+desc_option_luajit="Enable support for LuaJIT"
+build_options_default="tesseract wayland"
+
+# luajit is restricted to matching wordsize of host/target
+if [ "$XBPS_TARGET_WORDSIZE" = "$XBPS_WORDSIZE" ]; then
+	build_options_default+=" luajit"
+fi
+
+if [ "$CROSS_BUILD" ] && [ "$build_option_wayland" ]; then
+	hostmakedepends+=" wayland-devel"
```

Comment:
I think it's okay to make this a ` vopt_if` in `hostmakedepends` directly.

Reply:
indeed